packages/format-library/src/link/modal-screens/link-settings-screen.native.js

```diff
@@ -99,6 +102,7 @@ const LinkSettingsScreen = ( {
 		const format = createLinkFormat( {
 			url,
 			opensInNewWindow,
+			markedAsNoFollow,
```

Comment:
```suggestion
			markAsNoFollow: markedAsNoFollow,
```

Since the names are different we should update it so the right value gets to `createLinkFormat`